CanvasColorSpaceProposal.md

```diff
@@ -157,6 +157,17 @@ function encode(val) {
 </pre>
 * For all formats, the values stored in the framebuffer are in the canvas' color space.
 
+#### WebGPU behavior
+
+WebGPU's context configuration is specified dynamically using the `GPUCanvasContext` method `configureSwapChain`, which takes a `GPUSwapChainDescriptor` argument.
+Add an additional entry to `GPUSwapChainDescriptor` for the color space of the swap chain.
+<pre>
+partial dictionary GPUSwapChainDescriptor {
+  CanvasColorSpaceEnum colorSpace = "srgb";
+};
+</pre>
+Note that `GPUSwapChainDescriptor` already has a `texture` entry of type `GPUTextureFormat`.
```

Comment:
Probably worth explicitly pointing out that the `GPUTextureFormat` also defines the encoding (srgb vs non-srgb), if that's right.

Reply:
Oh, wait, GPUSwapChainDescriptor's IDL (that's checked in) has the entry `format`, not `texture` -- has that been changed?

Reply:
Reference: https://gpuweb.github.io/gpuweb/#dictdef-gpuswapchaindescriptor